src/app.js

```diff
@@ -98,10 +98,10 @@ export default function(app) {
   }
 
   function createElementFrom(node, isSVG) {
-    if (typeof node === "string") {
+    if (typeof node == "string") {
       var element = document.createTextNode(node)
     } else {
-      var element = (isSVG = isSVG || node.tag === "svg")
+      var element = (isSVG = isSVG || node.tag == "svg")
```

Comment:
This can technically fail because `tag` does not have to be a string (or it has to, up at this point? it can be a function at some point, but not sure if here we can guarantee that that's been resolved).

Anyway, here's the weird thing with `==` in case it's a function.

```
const foo = () => {}
foo.toString() // "() => {}"
foo == foo.toString() // true
```

However, `"svg"` is not a valid function, so I guess nothing bad can happen here.